messaging/src/main/java/org/axonframework/queryhandling/StreamingQueryMessage.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling;
+
+import org.axonframework.messaging.responsetypes.FluxResponseType;
+import org.axonframework.messaging.responsetypes.ResponseType;
+import reactor.core.publisher.Flux;
+
+import java.util.Map;
+
+/**
+ * A special type of {@link QueryMessage} used for initiating streaming queries. It is special since hard codes the
+ * response type to {@link FluxResponseType}.
```

Comment:
```suggestion
 * A special type of {@link QueryMessage} used for initiating streaming queries. It's special since it hard codes the
 * response type to {@link FluxResponseType}.
```